packages/cli/src/lib/cmds/play.ts

```diff
@@ -50,62 +50,52 @@ class Play {
   * @return {number} The largest version number found in the play console.
   */
   async getLargestVersion(twaManifest: TwaManifest): Promise<number> {
-    // TODO(@nohe427): This will return 0 until refactor finished.
-    const versionNumber = await this.googlePlay.getLargestVersionCode(twaManifest.packageId);
-    return versionNumber;
+    return await this.googlePlay.getLargestVersionCode(twaManifest.packageId);
   }
 
   /**
   * Publishes the Android App Bundle to the specified from the {@link PlayArgs}.
   * @return {boolean} Whether the publish command completes successfully or not.
   */
-  async publish(): Promise<boolean> {
+  async publish(twaManifest: TwaManifest): Promise<boolean> {
     // Validate that the publish value is listed in the available Tracks.
     // If no value was supplied with publish we make it internal.
     const userSelectedTrack = asPlayStoreTrack(this.args.publish?.toLowerCase() || 'internal');
     if (userSelectedTrack == null) {
       this.prompt.printMessage(enUS.messageInvalidTrack);
       return false;
     }
-    // appbundlelocation is an option argument.
-    if (this.args.appBundleLocation && fs.existsSync(this.args.appBundleLocation!!)) {
-      await this.googlePlay.publishBundleWithGradle(userSelectedTrack, this.args.appBundleLocation);
-      return true;
+
+    // Where we should find our default output file
```

Comment:
nit: I don't think this comment adds anything - the variable already has a pretty clear name.